app/workers/deletes.php

```diff
@@ -10,31 +10,43 @@
 use Appwrite\Database\Adapter\MySQL as MySQLAdapter;
 use Appwrite\Database\Adapter\Redis as RedisAdapter;
 use Appwrite\Database\Document;
+use Appwrite\Database\Validator\Authorization;
 use Appwrite\Storage\Device\Local;
+use Utopia\CLI\Console;
 use Utopia\Config\Config;
 
 class DeletesV1
 {
     public $args = [];
 
+    protected $consoleDB = null;
+
     public function setUp(): void
     {
     }
 
     public function perform()
     {
+        $projectId = $this->args['projectId'];
         $document = $this->args['document'];
+
         $document = new Document($document);
         
-        switch ($document->getCollection()) {
+        switch (strval($document->getCollection())) {
```

Comment:
You can also use:

```php
(string)$document->getCollection()
```

Just syntax sugar, don't have to change it.

Reply:
Sure @eldadfux